### ISSUE TITLE
Instruction how to read figshare h5 files

### ISSUE THREAD

**Issue by titipata** [ctn-archive/bekolay-jneurosci2014#1]
@tbekolay, do you have instruction on how to read `.h5` files from `figshare`. Currently, I use [`deepdish`](https://pypi.python.org/pypi/deepdish) commandline `ddls <file_name>.h5` in order to visualize the dataset. However, when I do

``` python
import deepdish as dd
data = dd.io.load('ctrl-adaptive-600.h5')
```

it gives me error as follows:

``` python
Too many indices for object '/Segment_0/analogsignals/AnalogSignal_0/sampling_rate'
```

**Comment by tbekolay** [comment]
The `h5` files were created with [python-neo](https://pythonhosted.org/neo/io.html#neo.io.NeoHdf5IO). They should be readable as general HDF5, but perhaps there's a bug in how neo writes HDF5 files? In any case, the relevant code for how I load the files is https://github.com/tbekolay/jneurosci2013/blob/master/scripts/sim_io.py#L201

**Comment by titipata** [comment]
I tried `neo` to read `h5` (with Python 3) as follows:

``` python
import neo.io
r = neo.io.NeoHdf5IO(filename='ctrl-adaptive-601.h5')
r.read_segment(path='/Segment_0')
```

However, I get the following error:

``` python
LookupError: The requested object with the path /Segment_0 exists, 
but is not of a NEO type. Please check the '_type' attribute.
```

Do you have any insight on that?

**Comment by tbekolay** [comment]
Hmm... I've not tried passing a `path` to `read_segment`, but my suspicion is that this is because the HDF5 file was saved with a previous version of `neo`. As is (fortunately!) recorded [here](https://github.com/tbekolay/jneurosci2013-bts/blob/master/02-virtualenv.ipynb), I saved the HDF5 files with `neo==0.3.1`, so try doing `pip install neo==0.3.1` and running your script again.

Also, I did all of this work in Python 2. I'm not sure if that would make any difference -- I hope not! But you never know...
